### ISSUE TITLE
Implement a way to get current legend/master Lost Sector

### ISSUE THREAD

**Issue by ghost** [Bungie-net/api#1325]
Beyond Light introduced rotating higher difficulty Lost Sectors (right now only on Europa & Cosmodrome, but who knows what the future will bring). They seem to switch daily, two out of the four a day (one with legend difficulty, one with master). They show up on the world map, and in-game can be launched with a flag post outside the lost sector entrance.

Unfortunately, there is no way to grab them via the API that I found. The problem is that they are not an "Activity" that can be launched from orbit (only those are returned from the CharacterActivities component 204), nor do they have a milestone/challenge or anything similar attached to them that would make them show up in CharacterProgressions 202.

In my opinion it would be best to have them (and honestly, all similar stuff you start in-game via flag pole and shows up on the world map) returned with CharacterActivities, and then maybe add an additional "canLaunchFromOrbit" property to that result.

**Comment by jshaffstall-bng** [comment]
Hopefully this is feasible. Filed as TFS 958683.

**Comment by kdeun1** [comment]
> Hopefully this is feasible. Filed as TFS 958683.

Hello. Lost sector information is not visible in characterActivities. (cf. `/Destiny2/${membershipType}/Profile/${membershipId}/Character/${characterId}/?components=204`)

How can I get the lost sector (legend, master) information? (i.e. Concealed Void: Legend (DestinyActivityDefinition hash: 912873277)

**Comment by lieuweberg** [comment]
The information is visible.
`/Platform/Destiny2/3/Profile/4611686018485161794/Character/2305843009534934620/?components=204`
(steam, my account, my hunter) for me returns
```
currentActivityHash:2829206727 // <Activity "K1 Communion: Legend">
currentActivityModeHash:547513715 // <ActivityMode "Scored Nightfall Strikes">
```
when I'm in the activity.
Bummer the activity mode is scored nightfall strikes, but that probably was the easiest to put there.

> The problem is that they are not an "Activity" that can be launched from orbit (only those are returned from the CharacterActivities component 204)

Non-Europa and Cosmodrome patrols do show, which I found really interesting when I found out. Can't come up with any other examples of this. (p.s. would love to see public events added ;) )
